demos-europe/demosplan-core
# Fix: setVisible is undefined error

Description: 
- if the baseLayer in enabled, that means not published, in public detail the console shows an error that setVisible has typeError undefined
- to not be confused, now there is a check in `Map.vue`, which shows a console.warn, that the baseLayer is not published yet, if firstTogglableLayer, which contains the attribute `setVisible`, is undefined

### How to review/test

- Different cases:
- 1.) Go to `Planungsdokumente und Planzeichnung`, click to `Kartenebenen definieren`, scroll to the baseLayers configuration and click `Bearbeiten`, activate the checkbox for `Karteneben veröffentlichen` and save this configuration. Go to the public detail and you will see the baseLayer and no console.warn and no typeError.
- 2.) Go to `Planungsdokumente und Planzeichnung`, click to `Kartenebenen definieren`, scroll to the baseLayers configuration and click `Bearbeiten`, deactivate the checkbox for `Karteneben veröffentlichen` and save this configuration. Go to the public detail and you will not see the baseLayer and a console.warn that the baseLayer is enabled and no typeError.

- Important: There is no request from the customer to this cases. So this is just a fix for the error in the console.
